samples/websockets/app.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: tornado
+  labels:
+    app: tornado
+spec:
+  ports:
+  - port: 8888
+    name: http
+  selector:
+    app: tornado
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: tornado
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: tornado
+        version: v1
+    spec:
+      containers:
+      - name: tornado
+        image: hiroakis/tornado-websocket-example
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 8888
+---
```

Comment:
Kill this

![image](https://user-images.githubusercontent.com/4461983/41524281-8e0a8aa2-730f-11e8-9bf4-3aec418f0909.png)